### DISCUSSION TITLE
Release a patch on crates.io to address breakages for non-MSVC Windows targets

### DISCUSSION THREAD

**Discussion by ratijas** [woboq/qmetaobject-rs#158] [General]
Can we release a patch bump, please? The bug #154 resolved in #155 unironically breaks my production build, and I don't like to patch creates.io with unversioned git stuff like `qttypes = { git = "https://github.com/woboq/qmetaobject-rs" }` in _Cargo.toml_. I mean, that's what I'm doing right now, but I won't commit it to the repo.

(Also, I'm testing this new shiny GitHub Discussions thingy)

**Comment by ogoffart** [comment]
I'll do a release

**Reply by ogoffart** [reply]
Done

**Reply by ratijas** [reply]
Thanks!

**Reply by ratijas** [reply]
Congrats on 300 stars, by the way :star: !
